yb-voyager/src/dbzm/dbzm.go

```diff
@@ -25,11 +25,12 @@ func NewDebezium(config *Config) *Debezium {
 
 func initVars() {
```

Comment:
Why not use `func init() {}`? As the `init()` functions get called automatically when a package is imported, you won't need to explicitly call the `initVars()`.

Reply:
oh ok sure, i didn't know about the concept of `init()` functions inside a package.